runtime/syntax/python.vim

```diff
@@ -80,6 +81,7 @@ syn keyword pythonRepeat	for while
 syn keyword pythonOperator	and in is not or
 syn keyword pythonException	except finally raise try
 syn keyword pythonInclude	from import
+syn keyword pythonAsync		async await
```

Comment:
If you take a look at the whole file you'll notice this at the end:
" vim:set sw=2 sts=2 ts=8 noet:
Therefore, the tabs *are* set to 8 (as they should) and the alignment works exactly as I intended.
This is a non-issue.
Thanks!